tests/unit/test_export.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import PIL
+
+from armory.utils.export import (
+    ImageClassificationExporter,
+    ObjectDetectionExporter,
+    VideoClassificationExporter,
+    VideoTrackingExporter,
+    So2SatExporter,
+)
+
+# Mark all tests in this file as `unit`
+pytestmark = pytest.mark.unit
+
+
+def test_image_classification_export():
+    random_x = np.random.rand(32, 32, 3)
+    exporter = ImageClassificationExporter(base_output_dir="/armory/tmp/")
+    pil_img = exporter.get_sample(random_x)
+    assert isinstance(pil_img, PIL.Image.Image)
+
+
+def test_object_detection_export():
+    random_x = np.random.rand(32, 32, 3)
+    exporter = ObjectDetectionExporter(base_output_dir="/armory/tmp")
+    pil_img = exporter.get_sample(random_x)
+    assert isinstance(pil_img, PIL.Image.Image)
+
+    y = {
+        "labels": np.array([1.0]),
+        "boxes": np.array([[0.0, 0.0, 1.0, 1.0]]).astype(np.float32),
+    }
+    y_pred = {
+        "scores": np.array([1.0]),
+        "labels": np.array([1.0]),
+        "boxes": np.array([[0.0, 0.0, 1.0, 1.0]]).astype(np.float32),
+    }
+
+    pil_img_with_both_boxes = exporter.get_sample_with_boxes(random_x, y_i=y, y_i_pred=y_pred)
+    assert isinstance(pil_img_with_both_boxes, PIL.Image.Image)
+
+    pil_img_with_gt_boxes = exporter.get_sample_with_boxes(random_x, y_i=y)
+    assert isinstance(pil_img_with_gt_boxes, PIL.Image.Image)
+
+    pil_img_with_pred_boxes = exporter.get_sample_with_boxes(random_x, y_i_pred=y_pred)
+    assert isinstance(pil_img_with_pred_boxes, PIL.Image.Image)
+
+
+
+def test_video_classification_export():
+    num_frames = 12
+    random_x = np.random.rand(num_frames, 32, 32, 3)
+    exporter = VideoClassificationExporter(base_output_dir="/armory/tmp", frame_rate=10)
+    pil_img_list = exporter.get_sample(random_x)
+    assert isinstance(pil_img_list, list)
+    assert len(pil_img_list) == num_frames
+    for img in pil_img_list:
+        assert isinstance(img, PIL.Image.Image)
+
+
+def test_video_tracking_export():
+    num_frames = 12
+    random_x = np.random.rand(num_frames, 32, 32, 3)
+    exporter = VideoTrackingExporter(base_output_dir="/armory/tmp", frame_rate=10)
+    pil_img_list = exporter.get_sample(random_x)
+    assert isinstance(pil_img_list, list)
+    assert len(pil_img_list) == num_frames
+    for img in pil_img_list:
+        assert isinstance(img, PIL.Image.Image)
+
+    y = {"boxes": np.ones((num_frames, 4)).astype(np.float32)}
+    y_pred = {"boxes": np.ones((num_frames, 4)).astype(np.float32)}
+
+    pil_img_with_boxes_list = exporter.get_sample_with_boxes(random_x, y, y_pred)
+
+    assert isinstance(pil_img_with_boxes_list, list)
+    assert len(pil_img_list) == num_frames
+    for img in pil_img_list:
+        assert isinstance(img, PIL.Image.Image)
+
+
+def test_so2sat_export():
+    random_x = np.random.rand(32, 32, 14)
+    exporter = So2SatExporter(base_output_dir="/armory/tmp")
+    sample_vh = exporter.get_vh_sample(random_x)
```

Comment:
what does `vh` mean?  Can we just keep `get_sample` as method name and introduce this as an arg where value can be `vh, vv, eo`?